### PR TITLE
feat: Adds support for super script [CW-1476]

### PR DESCRIPTION
**Pull request by nithindavid**
Parent issue - https://linear.app/chatwoot/issue/CW-1476/support-all-markdown-options-in-the-article-editor

https://user-images.githubusercontent.com/1277421/236614362-63db6ecc-f8c4-418f-83a2-17f3e4ed17a6.mov

### CODE REVIEW

**Comment by nithindavid** [comment]
I'll be adding a menu item for super and sub script

File: src/keymap.js [hunk 1]
```diff
@@ -111,6 +111,31 @@ export function baseKeyMaps(schema) {
...
+      //     marksToRemove.forEach(mark => {
+      //       tr.removeMark($cursor.pos - 1, $cursor.pos, mark);
+      //     });
+      //     if (dispatch) dispatch(tr);
+      //     return true;
+      //   }
+      // }
+      // return false;
+    });
+  }
```
Comment (pranavrajs) [inline src/keymap.js]: Please remove this if it is not used.

File: src/keymap.js [hunk 2]
```diff
@@ -74,6 +74,11 @@ export function baseKeyMaps(schema) {
     bind('Mod-I', toggleMark(schema.marks.em));
   }
 
+  if (schema.marks.superscript) {
+    bind('Mod-s', toggleMark(schema.marks.superscript));
+    bind('Mod-S', toggleMark(schema.marks.superscript));
```
Comment (scmmishra) [inline src/keymap.js]:
```suggestion
    bind('Mod-s', toggleMark(schema.marks.superscript));
    bind('Mod-S', toggleMark(schema.marks.superscript));
```

Can we make this `CMD Shift +` (Microsoft Word) or `CMD .` (Google Docs) instead. `CMD-S` is a known keyboard shortcut for "Save" action

Comment (nithindavid) [reply src/keymap.js]: Thanks for pointing this out; Will go with the google docs shortcut.

**Comment by nithindavid** [comment]
@scmmishra made it `Cmd+shift+ .` since the `cmd+ .` was not getting recorded. Let me know if this works

**Review by scmmishra — APPROVED** [review]

**Pull request merged**